src/main.rs

```diff
@@ -67,6 +67,9 @@ pub struct Cook {
     /// Build in release mode.
     #[clap(long)]
     release: bool,
+    /// Use nightly channel.
+    #[clap(long)]
+    nightly: bool,
```

Comment:
Let's leave channel selection outside of this PR, I'm looking to make it mirror what happens with `cargo` when installed via `rustup` - i.e. `cargo +nightly chef ...` should just work.

Reply:
Just rebased the PR to remove the relevant commit.